controllers/mytba_controller.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+import os
+
+from google.appengine.ext import ndb
+from google.appengine.ext.webapp import template
+
+from consts.model_type import ModelType
+from controllers.base_controller import LoggedInHandler
+from database.team_query import EventTeamsQuery
+from helpers.event_helper import EventHelper
+from helpers.event_team_status_helper import EventTeamStatusHelper
+from models.favorite import Favorite
+from models.team import Team
+
+
+class MyTBALiveController(LoggedInHandler):
+    def get(self):
+        self._require_login('/account/register')
+        self._require_registration('/account/register')
+
+        user = self.user_bundle.account.key
+        year = datetime.datetime.now().year
+        team_favorites_future = Favorite.query(Favorite.model_type == ModelType.TEAM, ancestor=user).fetch_async()
+
+        live_events = EventHelper.getWeekEvents()
+        favorite_team_keys = map(lambda f: ndb.Key(Team, f.model_key), team_favorites_future.get_result())
+        favorite_teams_future = ndb.get_multi_async(favorite_team_keys)
+
+        live_eventteams_futures = []
+        for event in live_events:
+            live_eventteams_futures.append(EventTeamsQuery(event.key_name).fetch_async())
+
+        favorite_teams = [team_future.get_result() for team_future in favorite_teams_future]
+        live_events_with_teams = EventTeamStatusHelper.buildEventTeamStatus(live_events, live_eventteams_futures, favorite_teams)
+
+        self.template_values.update({
+            'year': year,
```

Comment:
I believe you can just do event.year in the template instead of doing the datetime thing here.